mod-audit-server/src/main/java/org/folio/rest/impl/InitAPIs.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.*;
```

Comment:
Is code style applied?

Reply:
usage of wildcard imports is prohibited by some convention in Folio?

Reply:
@SerhiiNosko , actually, I don't see any strict rule for this https://dev.folio.org/guidelines/contributing/. But historically we try to skip wildcards. Definitely it's up to you. Just non-significant inline comments.